js/import/import.js

```diff
@@ -35,8 +35,12 @@ module.exports = function(moduleName, parentName) {
 			} else if(global.require){
 				resolve(global.require(moduleName));
 			} else {
-				// ideally this will use can.getObject
-				resolve();
+				try {
+					steal.import(moduleName, { name: parentName }).then(resolve, reject);
```

Comment:
anyway to avoid the try/catch? I think this should work...

```js
// steal slim
if(typeof stealRequire !== "undefined") {
  steal.import(moduleName, { name: parentName }).then(resolve, reject);
}
```